packages/icons-angular/examples/storybook/.storybook/config.js

```diff
@@ -0,0 +1,9 @@
+import { configure } from '@storybook/angular';
```

Comment:
Would it be helpful to add the story source addon? (Don't know if it applies for angular projects)

Also, for each icon could we generate three stories (or add equivalent documentation) for:

- default icon use case
- adding an a11y label
- specifying a title